app/streaming/session.cpp

```diff
@@ -982,6 +980,9 @@ void Session::exec(int displayOriginX, int displayOriginY)
         SDL_SetWindowFullscreen(m_Window, m_FullScreenFlag);
     }
 
+    // Pump the message loop to update the UI
+    emit connectionStarted();
```

Comment:
this is the main problem that application was crashed when starting streaming.
`window.visibility = false` would work when we apply only this part.
but exit stream still has problem.